public_api_client/public_api_client/configuration.py

```diff
@@ -0,0 +1,7 @@
+import json
+
+with open("/home/pib/public_api/config.json") as f: _data = f.read()
```

Comment:
env-variable is now used, if one is available

Reply:
Could you also add it to the docker-compose.yaml to the ros_packages service?

Reply:
env variable was added to docker-compose.yaml

Reply:
Ideally the keys are directly read from an env variable (and added to the password.env, each user has to create themselves). The docker container does not have the same file system as a our normal Ubuntu environment, because Docker containers typically don't have users.  If not done in this story, we should add a reminder for PR-604.

Reply:
the public-api-client now first checks, if the config file exists in the '/home/pib/...' directory and if this is not the case, the values are read from env variables.